### PR TITLE
added linting

### PR DESCRIPTION
**Pull request by atharvashirdhankar**
Added linting file

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/EventCalendar/pull/66"></a>

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - plugin:react/recommended
+  - plugin:json/recommended
+parserOptions:
+  ecmaFeatures:
+    jsx: true
+  ecmaVersion: 13
+  sourceType: module
+plugins:
+  - react
+rules: {
```
Comment (suitupstan) [inline .eslintrc.yml:15]: this is weird.

**Review by suitupstan — CHANGES_REQUESTED** [review]
Did you test running `npx eslint .`

File: .eslintrc.yml (hunk 1 above)
Comment (atharvashirdhankar) [reply .eslintrc.yml:15]: I run npx eslint .js file

**Comment by atharvashirdhankar** [comment]
Yes

**Comment by atharvashirdhankar** [comment]
npx eslint for seperate js file

**Comment by atharvashirdhankar** [comment]
It gave me some kind of errors in code .

**Review by suitupstan — CHANGES_REQUESTED** [review]
> It gave me some kind of errors in code .

Can those be fixed in this PR

**Comment by atharvashirdhankar** [comment]
Not actually in the code but it showed what changes can be done in the code.

**Comment by atharvashirdhankar** [comment]
Let me share the Screen shot.

File: .eslintrc.yml (hunk 1 above)
Comment (EmmaDawsonDev) [reply .eslintrc.yml:15]: @krishguptadev can you write a more helpful comment?

**Comment by atharvashirdhankar** [comment]
The output after running `npx eslint App.js` looks something like this 👇 

image

Sorry for the delay

**Comment by suitupstan** [comment]
Hey @StarTrooper08, I've opened a PR to your repository to fix all the issues (https://github.com/StarTrooper08/EventCalendar/pull/1). TIP: They could be fixed by running `eslint . --fix`. I made a PR since I wanted to introduce a few more changes 🙂

**Comment by atharvashirdhankar** [comment]
Ok using  eslint you also done the changes in existing reactjs code.
Nice
Actually for me Eslint is new to me but no issue learnt new thing during this contribution.

File: .eslintrc.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  'space-before-function-paren': ['error', 'never'],
+    'comma-dangle': ['error', 'always-multiline'],
```
Comment (naomi-lgbt) [inline .eslintrc.yml]:
```suggestion
  'space-before-function-paren': ['error', 'never'],
  'comma-dangle': ['error', 'always-multiline'],
```
These should have the same indent level, I believe.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: .eslintrc.yml (hunk 2 above)
Comment (atharvashirdhankar) [reply .eslintrc.yml]: Ok I will do the required changes

File: .eslintrc.yml (hunk 1 above)
Comment (suitupstan) [reply .eslintrc.yml:15]: There is json in b/w of yml? Is this normal?

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
This is blocked by the fact that the changes include a rollback of the `package-lock.json` file to `lockfileversion: 1`

**Pull request closed without merging**